include/wx/combo.h

```diff
@@ -585,6 +586,18 @@ class WXDLLIMPEXP_CORE wxComboCtrlBase : public wxControl,
 
     void OnPopupMouseEvent(wxMouseEvent& event);
 
+    // This function can be used as event handle for wxEVT_DPI_CHANGED event
+    // and simply recalculates button size when it happens.
+    void WXHandleDPIChanged(wxDPIChangedEvent& event)
```

Comment:
There doesn't seem to be any real reason to make this function inline. I've done it in a couple of classes which didn't have any corresponding `.cpp` file at all, but we do have one here and it would be better to move the function definition there.